pyro/nn/module.py

```diff
@@ -81,16 +82,21 @@ def _get_pyro_params(module):
 
 class PyroModule(torch.nn.Module):
     """
-    Subclass of :class:`torch.nn.Module` that supports setting of
-    :class:`PyroParam` and :class:`PyroSample` .
+    EXPERIMENTAL Subclass of :class:`torch.nn.Module` whose attributes can be
```

Comment:
does that mean we reserve the right to change the interface?

Reply:
Correct. I am especially uncertain about the synchronization policy between pyro param store and `PyroModule`.  We might soon declare this non-experimental in say pyro 1.1.